framework/src/main/java/org/apache/drill/test/framework/DrillTestDefaults.java

```diff
@@ -155,7 +155,12 @@
    */
   private static Map<String, String> getConfigProperties() {
     final Map<String, String> properties = Maps.newHashMap();
-    final File overrideFile = new File(CWD + "/conf/" + DRILL_TEST_CONFIG);
+
+    //CWD can be {basePath}/drill-test-framework or {basePath}/drill-test-framework/framework
+    final File overrideFile = new File(CWD +
+            (!CWD.endsWith("drill-test-framework") ? "/.." : "") +
```

Comment:
Do you want to make `drill-test-framework` a constant and have it in some place common with other such constants?

Reply:
@Agirish pointed out that sometimes the root directory could be renamed to something entirely different. So the logic here has to be more robust. Working on it.

Reply:
Reverting this fix for now as it affects only the changer for RM. This will unblock the PR from being merged.